kata-webhook/README.md

```diff
@@ -0,0 +1,57 @@
+# Kata Admission controller webhook
+
+Implement a simple admission controller webhook to annotate pods with the 
+kata runtime class.
+
+## How to use the admission controller
+
```

Comment:
Can we put a sentence here before we dive into the code block to explain what we are about to do - which is build the admission controller I think. The block title is a bit misleading - I thought it was going to explain how to install and use the controller (which is covered later on), but this section is just how to build the controller I think?

Reply:
Done